tools/retrieve_prs.py

```diff
@@ -19,10 +24,7 @@
 
 
 def _run_cmd(cmd):
-    try:
-        return subprocess.check_output(cmd).strip()
-    except Exception:
-        return None
+    return subprocess.check_output(cmd).decode('utf-8').strip()
```

Comment:
Note: This would fail if the commit message is not encoded with `utf-8`. (i.e. if user configures their git with non-uff commit message https://www.git-tower.com/help/guides/faq-and-tips/faq/encoding/windows)

The script needs to handle such case, while logging the failure and keep processing the rest of commits.

Reply:
Will follow-up the case in the future PR.